### PR TITLE
Set for code review

### PR DESCRIPTION
**Pull request by hsztan**
### First Project
Simple HTML and CSS webpage with a header that contains the text: "Hello Microverse!"
The project should have HTML and CSS files, including its respective linters.
A file called test.md should be created but ignored in .gitignore.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,3 +13,4 @@ <h1 class="greeting">Hello Microverse!</h1>
     </header>
   </body>
 </html>
+<!-- For code review -->
```
Comment (gemmen29) [inline index.html:16]: - Please add a description for the PR you can edit it by pressing the three dots icon then pressing edit 
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @hsztan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly push all your commits to the new feature branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (hsztan) [reply index.html:16]: @gemmen29 Thank you for your observations  💪

**Review by Wes-Isaac — APPROVED** [review]
Hi @hsztan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**